src/app/functions/server/onerep.ts

```diff
@@ -136,6 +144,25 @@ export async function createProfile(profileData: ProfileData): Promise<number> {
   return savedProfile.id;
 }
 
+export async function getProfile(
```

Comment:
Oh, I think I thought I needed this function, but then ended up not using it. Should I remove it, or shall I leave it in - it might come in handy later?

Reply:
I don’t have anything against leaving it as a helper function and think it might come in handy later.